packages/jest-jasmine2/src/reporter.ts

```diff
@@ -6,7 +6,7 @@
  */
 
 import {Config} from '@jest/types';
-import {AssertionResult, TestResult} from '@jest/test-result';
+import {AssertionResult, TestResult, createEmptyTestResult} from '@jest/test-result';
```

Comment:
prettier wants you to format this

Reply:
Hmm, I ran the lint with `yarn lint:prettier`, not sure why it wasn't caught

Reply:
that only runs on `md` files - you want `yarn lint` which runs eslint that we've configured to use with prettier